### DISCUSSION TITLE
Winglight off

### DISCUSSION THREAD

**Discussion by crazyrolf** [flybywiresim/aircraft#3595] [Ideas]
It would be good if the winglight did not always go on. 
Normally, you use it only rarely, in the dark and when there is a risk of ice.

Captain Rudolfo

**Comment by hotshotp** [comment]
you can... turn it off? its a switch

**Comment by crazyrolf** [comment]
really …… zzzzz
standard has to be off.

**Reply by hotshotp** [reply]
its off... what do you mean? its a switch, you use it like any other light...

**Comment by crazyrolf** [comment]
when you start on the runway, it is always on. It should be off.
Even if you start in the air, approaching an airport it is always on, it should be off.

CR

**Reply by hotshotp** [reply]
well starting in the air or the runway isn't very realistic anyway tbh, not really that big of an issue
